test/functional/pdo_sqlsrv/pdo_exec_invalid_query.phpt

```diff
@@ -1,29 +1,31 @@
 --TEST--
 direct execution of an invalid query
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-    require_once("MsSetup.inc");
+    require_once("MsCommon_mid-refactor.inc");
+
+    $conn = connect();
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_SILENT);
+
```

Comment:
again, why SILENT mode?

Reply:
The original test prints the SQLSTATE

Reply:
I see... SILENT is the default exception mode